cmd/bacalhau/job-test.yaml

```diff
@@ -0,0 +1,60 @@
+# jobspec for the bacalhau command bacalhau docker run ubuntu -- /bin/bash -c 'echo "simple job output"'
+apiVersion: v1alpha1
+engine: 2
```

Comment:
Might want to add a `kind` to this, to allow for extension of other things later. E.g. a "dag of job specs" or whatever.